react-backend/routes/signup.js

```diff
@@ -0,0 +1,46 @@
+var express = require("express");
+var router = express.Router();
+var mysql = require("mysql");
+var config = require("../config");
+var bodyParser = require("body-parser");
+
+const connection = mysql.createPool({
+  host: config.host,
+  user: config.user,
+  password: config.password,
+  database: config.database
+});
+
+router.post("/", function(req, res, next) {
+  var accountID = req.query.accountID;
+  var email = req.query.email;
+  var password = req.query.password;
+  var balance = req.query.balance;
+  
+  query =
```

Comment:
Nice sql injection prevention! I'll have to adapt my queries to do the same!

Reply:
I have fixed all the other queries in the encryption branch so no need to worry about it!